ocelli-core/src/main/java/netflix/ocelli/loadbalancer/SettableLoadBalancer.java

```diff
@@ -0,0 +1,28 @@
+package netflix.ocelli.loadbalancer;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicReference;
+
+import netflix.ocelli.LoadBalancer;
+import rx.functions.Action1;
+
+/**
+ * Base for any {@link LoadBalancer} whose pool of available targets is tracked as an immutable
+ * list of T that can be updated at any time.  A SettableLoadBalancer is meant to be the final
+ * subscriber to an RxJava stream that manages lifecycle for targets.
+ * 
+ * @author elandau
+ *
+ * @param <T>
+ */
+public abstract class SettableLoadBalancer<T> extends LoadBalancer<T> implements Action1<List<T>> {
```

Comment:
I think this will come in to play when we need the abstraction for setting the current active list of hosts.  For example, we may have a builder for setting up the load balancing strategy.  We could move the atomic reference into the concrete implementation and keep SettableLoadBalancer as an interface.